services/workflows-service/src/alert/alert.controller.external.ts

```diff
@@ -135,7 +137,17 @@ export class AlertControllerExternal {
   async decision(
     @common.Body() { alertIds, decision }: AlertDecisionDto,
```

Comment:
i saw that you've change the AlertDecisionDto which isnt extend from AlertsIdsByProjectDto than you dont have alertsIds this property isnt exists on the DTO

Reply:
I do. I copied over `alertIds`. We can't extend that DTO because it forces a project ID with a min length of 1.

Reply:
yeah sound good, projectId is required fields and it should be string empty

Reply:
Can we resolve this comment?